nixnet/session.py

```diff
@@ -25,6 +25,35 @@
 
 
 class Session(object):
+    """What is a Session?
+
+    The NI-XNET session represents a connection between your National Instruments
+    CAN/FlexRay/LIN hardware and hardware products on the external network.
+
```

Comment:
The first-line of a docstring should be a summary that stands on its own.  "What is a Session?" does not meet that criteria